addon/components/liquid-container.js

```diff
@@ -9,8 +9,8 @@ export default Component.extend(Growable, {
   classNames: ['liquid-container'],
 
   lockSize(elt, want) {
-    elt.outerWidth(want.width);
-    elt.outerHeight(want.height);
+    elt.style.width = want.width;
+    elt.style.height = want.height;
```

Comment:
An example of what I'm talking about: 

This code is not equivalent. It's going to introduce animation-jumping bugs in some situations that have padding and/or borders.

Reply:
Understood. I'll cease this effort.